src/sinks/elasticsearch/sink.rs

```diff
@@ -67,7 +68,12 @@ where
                 future::ready(Some(match event {
                     Event::Metric(metric) => metric_to_log.transform_one(metric),
                     Event::Log(log) => Some(log),
-                    _ => None,
+                    Event::Trace(_) => {
+                        // Although technically this will cause the event to be dropped, due to the sink
+                        // config it is not possible to send traces to this sink - so this situation can
+                        // never occur. We don't need to emit an events_dropped event.
```

Comment:
Not really an issue but I've been trying to refer to it as  " `EventsDropped` internal event " , in comments to correspond to the instrumentation spec doc ?